src/shared/pages/team/addToTeam/addToTeam.tsx

```
@@ -34,17 +34,15 @@ const AddToTeam: React.FC<AddToTeamProps> = ({ history, match }) => {
             return;
         }
 
-        Promise.all(state.selectedUsers.map(user =>
-            addUserToTeam(user, teamId)
-                .then(() => dispatch({ type: 'RemoveFromSelection', payload: user }))
-                .catch((error: AddUserError) => {
-                    const { userToAdd: { label, value } } = error;
-                    addFormError({ key: value, value: label });
-                    throw error;
-                })
-        )).then(() => {
-            history.push(`/team-view/${teamId}`, { successMessage: ADD_USER_SUCCESS });
-        }).catch(() => { });
+        addUsersToTeam(state.selectedUsers, teamId)
+            .then(() => dispatch({ type: 'RemoveAllFromSelection' }))
+            .catch((error: AddUserError) => {
+                error.usersToAdd.map(({ label, value }) => addFormError({ key: value, value: label }));
+                throw error;
+            })
+            .then(() => {
+                history.push(`/team-view/${teamId}`, { successMessage: ADD_USER_SUCCESS });
+            }).catch(() => { });
```

Can `history.push` throw? If not can we remove this empty catch?